<launch>
  <arg name="namespace"             default="/"/>
  <arg name="image_topic_in"        default="/stereo/left/image_rect"/>
  <arg name="image_topic_out"       default="/stereo/left/sync/image_rect"/>
  <arg name="camera_info_topic_in"  default="/stereo/left/camera_info"/>
  <arg name="camera_info_topic_out" default="/stereo/left/sync/camera_info"/>
  <!-- ===================================================================== -->
  <node pkg="apriltag_sync" exec="apriltag_sync_node" name="apriltag_sync_node" namespace="$(var namespace)" output="screen">
    <param name="image_topic_in"        value="$(var image_topic_in)"/>
    <param name="image_topic_out"       value="$(var image_topic_out)"/>
    <param name="camera_info_topic_in"  value="$(var camera_info_topic_in)"/>
    <param name="camera_info_topic_out" value="$(var camera_info_topic_out)"/>
  </node>
</launch>
